openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -6339,21 +6340,24 @@ OS:Foundation:Kiva,
        \type real
        \minimum> 0.0
        \default 0.3
+  N10, \field Number of Custom Blocks
+       \minimum 0.0
```

Comment:
Shouldn't this be formatted as an integer (no decimal point), also can be autocalculated.  See OS:Surface Number of Vertices

Reply:
I was using `OS:Generator:FuelSupply` as a guide. It has `\minimum=0.0` and `\maximum=12.0`. But yes I will take a look at `OS:Surface` Number of Vertices.

Reply:
Decided to remove this field from the idd.